examples/connect-latency.yaml

```diff
@@ -0,0 +1,21 @@
+metrics:
+  histograms:
+    - name: connect_latency_seconds
+      help: Latency histogram for TCP connect() syscall
```

Comment:
If it's TCP only, then let's call the metric `tcp_connect_latency_seconds`  and rename the file into `tcp-connect-latency.yaml`.

Reply:
Since we do this:

```
// Filter out non-blocking sockets and errors
```

It's probably a good idea to add `blocking` in the name as well.